spec/schema.json

```diff
@@ -0,0 +1,32 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Companies House - Limited Partnerships API",
+    "description": "An API to allow the registration, transition and update of limited partnerships",
+    "version": "1.0.0"
+  },
+  "host": "api.companieshouse.gov.uk",
+  "tags": [
+    {
+      "name": "Limited Partnerships"
+    }
+  ],
+  "paths": {
```

Comment:
Ideally the initial spec should just have a healthcheck endpoint which is `/limited_partnership/healthcheck` according to the service design doc: https://companieshouse.atlassian.net/wiki/spaces/Arch/pages/4722491429/LP+Service+Design#:~:text=/limited_partnership/healthcheck

Reply:
Updated spec endpoint using a hyphen for limited-partnership.